docs/reference/query-dsl/query-string-query.asciidoc

```diff
@@ -63,6 +63,8 @@ with default operator of `AND`, the same query is translated to
 
 |`analyzer` |The analyzer name used to analyze the query string.
 
+|`quote_analyzer` |The analyzer name used to analyze the quote part of query string.
```

Comment:
Looks okay, but reading this I still don't get what the parameter is used for. Were you able to find out what this parameter is actually doing? I'm asking because I was just browsing the documentation and other resources on the web (girthub, the forum) and couldn't find any reference except in our own source code and in the javadoc of `QueryStringQueryBuilder`.